xpdacq/beamtime.py

```diff
@@ -601,6 +601,11 @@ def md(self):
         """ metadata of current object """
         return dict(self)
 
+    @property
+    def robot_all(self):
+        """All samples in the robot magazine"""
```

Comment:
I guess this property is to show all samples in the magazine. Could you change to something like `show_all_sample_in_magzine` or other better name?

Reply:
Sure

Reply:
I removed the show since the primary intent of this was to be used in `xrun` as a place holder for all the samples (eg, run a robot scan on all the samples)

Reply:
sure. makes sense.